pkgs/top-level/linux-kernels.nix

```diff
@@ -343,7 +343,7 @@ in {
 
     dddvb = callPackage ../os-specific/linux/dddvb { };
 
-    decklink = callPackage ../os-specific/linux/decklink { };
+    decklink = throw "The decklink kernel module has been removed due to being unmaintained"; # Module removed on 2024-07-02
```

Comment:
Why do you remove the `blackmagic-desktop-video` completely. Maybe just remove yourself from the maintainers list there.
This would allow other people to step in; in case anything breaks in the future or if they want?

Reply:
As already stated in the commit message: the package is quite difficult to maintain, complex, and breaks with basically every major kernel update or software update by BMD. The last time i (temporarily) stepped back from nixpkgs the package turned unusable within a few months.

Reply:
@hsjobeki consider next time sending a PR to bring back a package rather than taking this approach in such PRs. This is a matter of respect.